internal/staging/start_testrunner.go

```diff
@@ -341,14 +341,15 @@ func (c *controller) sendTestPendingResult(queue *s2hv1.Queue) error {
 		return err
 	}
 
+	var error error
```

Comment:
why can't we use `err`?

Reply:
I want to return the error from the function in for loop scope so, I handle it separately.
but i double checked on it it can be reuse, already fixed